tests/FSharp.Compiler.ComponentTests/FSharpChecker/TransparentCompiler.fs

```diff
@@ -894,4 +895,49 @@ let ``LoadClosure for script is recomputed after changes`` () =
         |> Seq.map (fun (k, g) -> k, g |> Seq.map fst |> Seq.toList)
         |> Map
 
-    Assert.Equal<JobEvent list>([Weakened; Requested; Started; Finished; Weakened; Requested; Started; Finished], closureComputations["FileFirst.fs"])
\ No newline at end of file
+    Assert.Equal<JobEvent list>([Weakened; Requested; Started; Finished; Weakened; Requested; Started; Finished], closureComputations["FileFirst.fs"])
+    
+[<Fact>]
+let ``TryGetRecentCheckResultsForFile returns None before first call to ParseAndCheckFileInProject`` () =
+    let project = SyntheticProject.Create(
+        sourceFile "First" [])
+    
+    ProjectWorkflowBuilder(project, autoStart = false, useTransparentCompiler = true) {
```

Comment:
Also, should we run these also for `BackgroundCompiler`? If you remove `useTransparentCompiler = true` the CI will run both. And locally you can set `FSHARP_EXPERIMENTAL_FEATURES=1` env variable to be testing with TC.

Reply:
done